push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/apns/PushyApnsSender.java

```diff
@@ -109,7 +109,7 @@ public void sendPushMessage(final Variant variant, final Collection<String> toke
             }
         }
 
-        if (apnsClient.isConnected()) {
+        if (apnsClient != null && apnsClient.isConnected()) {
```

Comment:
why the null check? It looks like `receiveApnsConnection` can either return a valid client or throw.